contrib/add-copyright.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Simple script to add SPDX copyright notice header lines to new files
+# using reuse.
+#
+# The script expects at least one argument (i.e., the FILE).
+# It can accept more arguments, that will be passed as CLI options
+# to reuse, but does not check for the validity of any argument.
+# Therefore wrong arguments may trigger reuse errors.
+#
+# Default options:
+# - Use 'Red Hat, Inc.' as default Copyright holder.
+# - GPL-2.0-or-later as default license for new files in the project.
+# - Use vdsm-specific notice template.
+# - Exclude years, as they are not required and makes it easier to mantain.
+#
+# Usage:
+# ./contrib/add-copyright.sh [OPTIONS] FILE
+# Examples:
+# ./contrib/add-copyright.sh new_file.py
+# ./contrib/add-copyright.sh --style html new_file.html
+
+
+if [ -z "$1" ]; then
+    echo "ERROR: File missing"
+    echo "Usage:"
+    echo "  $0 [OPTIONS] FILE"
+    exit 1
+fi
+
+reuse addheader \
+    --copyright="Red Hat, Inc." \
+    --license="GPL-2.0-or-later" \
```

Comment:
IMO the script covers the (by far) most common case, and it is good as `add-spdx-header.sh`. There are only a handful of files that use a different license in the project. If we need to add another file with an alternate license, we can rely on the `reuse` documentation, or document how to do it in `doc/development.md`.

Also, the script itself is simple and contains the command that would be used in a clear way. So it can be read to get the command syntax.

Reply:
You can include this info in the script header, and explain how to handle files that need another license.